Gemfile.lock

```diff
@@ -101,6 +101,7 @@ GEM
       factory_girl (~> 4.7.0)
```

Comment:
Removed these changes to Gemfile.lock caused by the windows environment, excluding the additions of redis (as these should have been generated due to the redis gem regardless of environment).